src/display-buffer.coffee

```diff
@@ -37,7 +37,7 @@ class DisplayBuffer extends Model
 
   verticalScrollMargin: 2
   horizontalScrollMargin: 6
-  scopedCharacterWidthsChangeCount: 0
+  characterWidthsDidChange: false
```

Comment:
Maybe `characterWidthsChanged`? We have been using `will` and `did` in the names of hooks.